### DISCUSSION TITLE
Use pkgx to load python 3

### DISCUSSION THREAD

**Discussion by acgetchell** [pkgxdev/pkgx#795] [Help]
I'm trying out a Rust library that uses `pyplot`, am I correct in thinking that I should be able to use `pkgx` to load Python3 for me in the repo directory?

If so, I have a parse error.

https://github.com/pkgxdev/pkgx/issues/794

If not, what's the error in my mental model?

Thanks,
Adam

**Comment by mxcl** [comment]
yes, provided you have shell integration, you can either do `env +python3` or add `python3` to the `dev` via keyfiles: https://docs.pkgx.sh/dev
